### PR TITLE
adding win_prefix_path

### PR DESCRIPTION
**Pull request by luthermonson**
@rawmind0 i have no idea if this is right feel free to shoot holes in it

### CODE REVIEW

File: rancher2/schema_cluster_rke_config.go [hunk 1]
```diff
@@ -137,6 +137,12 @@ func clusterRKEConfigFieldsV0() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "Optional prefix to customize kubernetes path",
 		},
+		"win_prefix_path": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+			Description: "Optional prefix to customize kubernetes path for windows",
+		},
```
Comment (rawmind0) [inline rancher2/schema_cluster_rke_config.go]: This added section is not needed here due to `func clusterRKEConfigFieldsV0()` is referencing old tf schema

File: rancher2/structure_cluster_rke_config.go [hunk 2]
```diff
@@ -122,6 +122,10 @@ func flattenClusterRKEConfig(in *managementClient.RancherKubernetesEngineConfig,
 		obj["prefix_path"] = in.PrefixPath
 	}
 
+	if len(in.PrefixPath) > 0 {
```
Comment (rawmind0) [inline rancher2/structure_cluster_rke_config.go]: typo here, should check `in.WindowsPrefixPath`

```
-	if len(in.PrefixPath) > 0 {
+	if len(in.WindowsPrefixPath) > 0 {
```

File: rancher2/structure_cluster_rke_config_test.go [hunk 3]
```diff
@@ -57,6 +57,7 @@ func testClusterRKEConfig() {
 			"network":               testClusterRKEConfigNetworkInterfaceCanal,
```
Comment (rawmind0) [inline rancher2/structure_cluster_rke_config_test.go:58]: `WindowsPrefixPath` field should also be added to `testClusterRKEConfigConf` struct

```
...
	testClusterRKEConfigConf = &managementClient.RancherKubernetesEngineConfig{
...
		PrefixPath:          "terraform-test",
+		WindowsPrefixPath:   "c:\\terraform-test",
		PrivateRegistries:   testClusterRKEConfigPrivateRegistriesConf,
...
```

**Review by rawmind0 — CHANGES_REQUESTED** [review]
@luthermonson , thanks for the PR

Please, take a look to changes requested

**Comment by luthermonson** [comment]
@rawmind0 updated!

**Review by rawmind0 — APPROVED** [review]

**Comment by rawmind0** [comment]
LGTM

**Pull request merged**